onnxruntime/test/python/transformers/test_flash_attn.py

```diff
@@ -1141,84 +1159,165 @@ def parity_check_gqa_past_no_buff(
     )
 
 
+class TestMHA(unittest.TestCase):
+    def test_packed_mha(self):
+        major, _ = torch.cuda.get_device_capability()
+        if major < 8:
+            return
+        print("-------- TEST PACKED MHA ---------")
+        for b in [5]:
```

Comment:
why choose these number?
For example, may add boundary case s=1; common h = 16.